compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1126,9 +1126,32 @@ static SmallVector<int64_t> getLinalgExtDefaultWorkgroupTileSizes(
   return workgroupTileSizes;
 }
 
+static LogicalResult setTransformStrategyRootConfig(func::FuncOp entryPointFn,
+                                                    tensor::PackOp op) {
+  if (!clCPUEnableTransformDialectJit) return failure();
+
+  // TODO(hanchung): Move the checks to matchAndSetPackStrategy. We can't move
+  // them to the method because of cyclic deps.
+  auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(entryPointFn);
+  if (!isX86_64(targetAttr)) return failure();
+
+  cpu::CPUModel cpuModel{.lowerToAVX2 = hasAVX2Feature(targetAttr)};
```

Comment:
this breaks msvc

Reply:
Do you checkout to the PR and run tests? The PR needs an upstream commit, which is not cherry-picked into IREE yet. Let me cherry-pick it in the PR and you can give it another shot.